### ISSUE TITLE
Unexpectedly absent the warning marker in the editor

### ISSUE THREAD

**Issue by artaleks9** [eclipse-che/che#7161]
* Sometimes selenium tests from 'opendeclaration' package are failed by reason not present warning markers
* Another case of failing test: after perform the 'open declaration' (F4) the expected file is absent in the editor
* As a rule it happen when tests were running on CI, but on local host the tests are passed
* We need to fix up it
* Related selenium tests: _Eclipse0093Test_, _Eclipse0115Test_, _Eclipse0121Test_ , _AutocompleteProposalJavaDocTest_ , _AutocompleteWithInheritTest_ , _ShowHintsCommandTest_

**Comment by artaleks9** [comment]
After merging the pull request #7164 the problem is not reproduced on CI

**Comment by dmytro-ndp** [comment]
We have the same test failure again on CI https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/113/testReport/junit/org.eclipse.che.selenium.opendeclaration/Eclipse0115Test/test0115/:
```
org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: //div[@class='ruler overview']/div[@class='annotationOverview warning'] (tried for 20 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'slave6.codenvycorp.com', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.10.0-514.16.1.el7.x86_64', java.version: '1.8.0_102'
Driver info: org.eclipse.che.selenium.core.SeleniumWebDriver
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:265)
	at org.eclipse.che.selenium.pageobject.CodenvyEditor.waitMarkerInPosition(CodenvyEditor.java:627)
```
![org eclipse che selenium opendeclaration eclipse0115test test0115_qqldkvfe](https://user-images.githubusercontent.com/1197777/33932335-61a70794-dffb-11e7-9b22-44133d0ba3a2.png)

- https://ci.codenvycorp.com/view/pr-builds/job/che-pullrequests-test/279/testReport/org.eclipse.che.selenium.opendeclaration/Eclipse0115Test/test0115/

**Comment by dmytro-ndp** [comment]
Another unstable test: org.eclipse.che.selenium.opendeclaration.Eclipse0121Test.test0121():
- https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/113/testReport/junit/org.eclipse.che.selenium.opendeclaration/Eclipse0121Test/test0121/
- https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/95/testReport/junit/org.eclipse.che.selenium.opendeclaration/Eclipse0121Test/test0121/
```
  org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of all elements located by By.xpath: //div[@id='gwt-debug-multiSplitPanel-tabsPanel']//div[text()='Collections'] (tried for 10 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'slave6.codenvycorp.com', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.10.0-514.16.1.el7.x86_64', java.version: '1.8.0_102'
Driver info: org.eclipse.che.selenium.core.SeleniumWebDriver
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:265)
	at org.eclipse.che.selenium.pageobject.CodenvyEditor.waitTabIsPresent(CodenvyEditor.java:1114)
	at org.eclipse.che.selenium.opendeclaration.Eclipse0121Test.test0121(Eclipse0121Test.java:60)
```
![org eclipse che selenium opendeclaration eclipse0121test test0121_61hlm96w](https://user-images.githubusercontent.com/1197777/33940969-bb813898-e019-11e7-90aa-90422c46e756.png)

**Comment by artaleks9** [comment]
Right now the problem is not reproduced on CI

**Comment by artaleks9** [comment]
The problem is again actual, so the issue is reopened and delegate to developer team

**Comment by artaleks9** [comment]
The working of product  is OK from developer side
And it resolved by the #8797
